terraform/binary_path.go

```diff
@@ -1,22 +1,21 @@
 package terraform
 
 import (
+	"embed"
+	"errors"
 	"fmt"
 	"os"
-	"path/filepath"
 	"strconv"
 	"strings"
 	"time"
 
-	"github.com/gobuffalo/packr/v2"
 	"github.com/spf13/afero"
 )
 
 const (
 	tfBinDataAssetName = "terraform"
 	bblTfBinaryName    = "bbl-terraform"
-	terraformBinary    = "./binary_dist"
```

Comment:
Why was this removed?

Reply:
this is removed as it was a constant that was used within the packr code block